src/Plugins/Tests.Security/EndToEndTests.cs

```diff
@@ -58,6 +58,8 @@ protected EndToEndTests(ITestOutputHelper outputHelper) : base(outputHelper)
 
         protected override string TestLogResourceNameRoot => $"Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.TestData.{TypeUnderTest}";
 
+        protected override string ProductRootDirectory => Path.Combine(ThisAssembly.Location, "..", "..", "..", "..", "..", "..");
```

Comment:
>ProductRootDirectory [](http://example.com/codeflow?start=34&length=20)

We have a `git.exe` from SDK and we use it to get various path, which will prevent our code running in Windows, here I get away with overwrite it.

Reply:
Hm, ok. This could break some other environment that produces a path, but we will see! :)